internal/sdk/cloudian/sdk.go

```diff
@@ -230,6 +236,43 @@ func (client Client) CreateUser(ctx context.Context, user User) error {
 	return resp.Body.Close()
 }
 
+// Fetch user credentials.
+func (client Client) GetUserCredentials(ctx context.Context, user User) ([]SecurityInfo, error) {
```

Comment:
I think the go convention is to (ideally) comment all exposed functions.  
If we want to follow it, we should also follow the convention of starting with the function name.

https://tip.golang.org/doc/comment#func

```suggestion
// GetUserCredentials fetches all the credentials of a user.
func (client Client) GetUserCredentials(ctx context.Context, user User) ([]SecurityInfo, error) {
```